File: Problem001/sol.c
#include <stdio.h>

int main (void) {
    int result = 0;
    int i;
    
    for(i = 0; i < 1000; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            result += i;
        }
    }
    
    printf("%d\n", result);
    
    return 0;
}

File: Problem001/sol.cs
using System;

class Program {
    static void Main() {
        int result = 0;
    	
        for (int num = 0; num < 1000; num++) {
            if (num % 3 == 0 || num % 5 == 0) {
                    result += num;
            }
        }
		
        Console.WriteLine(result);
    }
}
